Shared/Controls/ImageSlideshow/ImageSlideshow.cs

```diff
@@ -1,13 +1,19 @@
-﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS10_0_18362_0_OR_GREATER
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
 using Uno.Extensions;
 using Uno.Extensions.Specialized;
+#if WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+#else
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Controls.Primitives;
+#endif
 using System.Collections.Specialized;
 using Uno.Disposables;
 using Windows.UI.Core;
```

Comment:
For WinUI should be using DispatcherQueue (Microsoft.Windows.Dispatching) instead of CoreDispatcher (Windows.UI.Core)